app/src/ui/dialog/dialog.tsx

```diff
@@ -3,6 +3,8 @@ import classNames from 'classnames'
 import { DialogHeader } from './header'
 import { createUniqueId, releaseUniqueId } from '../lib/id-pool'
 import { getTitleBarHeight } from '../window/title-bar'
+import { clamp } from '../../lib/clamp'
+import { infinity } from '../octicons/octicons.generated'
```

Comment:
I think this is not the `infinity` you were looking for

![These are not the droids you're looking for](https://user-images.githubusercontent.com/1083228/196920533-d48926f8-a699-4c51-baca-2df9705366c8.gif)

Reply:
haha. Yes. Surprised it worked..